json/src/main/scala/com/gu/contentapi/json/Serialization.scala

```diff
@@ -0,0 +1,233 @@
+    case JField("shouldHideAdverts", JString(s)) => JField("shouldHideAdverts", JBool(s.toBoolean))
+    case JField("hasStoryPackage", JString(s)) => JField("hasStoryPackage", JBool(s.toBoolean))
+    case JField("isExpired", JString(s)) => JField("isExpired", JBool(s.toBoolean))
+    case JField("syndicatable", JString(s)) => JField("syndicatable", JBool(s.toBoolean))
+    case JField("subscriptionDatabases", JString(s)) => JField("subscriptionDatabases", JBool(s.toBoolean))
+    case JField("developerCommunity", JString(s)) => JField("developerCommunity", JBool(s.toBoolean))
+    case JField("commentable", JString(s)) => JField("commentable", JBool(s.toBoolean))
+    case JField("liveBloggingNow", JString(s)) => JField("liveBloggingNow", JBool(s.toBoolean))
+    case JField("isPremoderated", JString(s)) => JField("isPremoderated", JBool(s.toBoolean))
+    case JField("wordcount", JString(s)) => JField("wordcount", JInt(s.toInt))
+    case JField("newspaperPageNumber", JString(s)) => JField("newspaperPageNumber", JInt(s.toInt))
+    case JField("starRating", JString(s)) => JField("starRating", JInt(s.toInt))
+    case JField("isInappropriateForSponsorship", JString(s)) => JField("isInappropriateForSponsorship", JBool(s.toBoolean))
+    case JField("isInappropriateForAdverts", JString(s)) => JField("isInappropriateForAdverts", JBool(s.toBoolean))
+    case JField("internalPageCode", JString(s)) => JField("internalPageCode", JInt(s.toInt))
+    case JField("internalStoryPackageCode", JString(s)) => JField("internalStoryPackageCode", JInt(s.toInt))
+    case JField("width", JString(s)) => JField("width", JInt(s.toInt))
+    case JField("height", JString(s)) => JField("height", JInt(s.toInt))
+    case JField("duration", JString(s)) => JField("duration", JInt(s.toInt))
+    case JField("durationMinutes", JString(s)) => JField("durationMinutes", JInt(s.toInt))
+    case JField("durationSeconds", JString(s)) => JField("durationSeconds", JInt(s.toInt))
+    case JField("isMaster", JString(s)) => JField("isMaster", JBool(s.toBoolean))
+    case JField("sizeInBytes", JString(s)) => JField("sizeInBytes", JLong(s.toLong))
+    case JField("blockAds", JString(s)) => JField("blockAds", JBool(s.toBoolean))
+    case JField("allowUgc", JString(s)) => JField("allowUgc", JBool(s.toBoolean))
+    case JField("displayCredit", JString(s)) => JField("displayCredit", JBool(s.toBoolean))
+    case JField("legallySensitive", JString(s)) => JField("legallySensitive", JBool(s.toBoolean))
+    case JField("sensitive", JString(s)) => JField("sensitive", JBool(s.toBoolean))
+    case JField("explicit", JString(s)) => JField("explicit", JBool(s.toBoolean))
+    case JField("clean", JString(s)) => JField("clean", JBool(s.toBoolean))
+    case JField("safeEmbedCode", JString(s)) => JField("safeEmbedCode", JBool(s.toBoolean))
+  }
+
+  def thriftEnum2JString[A <: ThriftEnum]: PartialFunction[Any, JString] = {
+    case a: ThriftEnum => JString(pascalCaseToHyphenated(a.name))
```

Comment:
The reason is that the old behaviour was wrong :)

When CAPI renders an enum value in JSON, it writes it in lowercase hyphenated form (e.g. "newspaper-book-section"). And we need to maintain that behaviour so we don't break any clients who use JSON. But the equivalent Thrift enum name is e.g. `NewspaperBookSection`.